core/templates/pages/story-viewer-page/story-viewer-page.directive.html

```diff
@@ -1,255 +1,90 @@
-                 style="position:relative">
-              <defs>
-                <pattern id="image<[$index]>" patternUnits="userSpaceOnUse" height="150" width="100">
-                  <circle cx="50"
-                          cy="70"
-                          r="50"
-                          ng-attr-fill="<[$ctrl.pathIconParameters[$index].thumbnailBgColor]>"/>
-                  <image ng-if="node.isCompleted()"
-                         x="0"
-                         y="20"
-                         height="100"
-                         width="100"
-                         ng-attr-xlink:href="<[$ctrl.getStaticImageUrl('/general/collection_paw.svg')]>"
-                         xlink:href=""
-                         transform="rotate(-75 50 70)">
-                  </image>
-                  <image ng-if="!node.isCompleted()"
-                         x="10"
-                         y="30"
-                         height="80"
-                         width="80"
-                         ng-attr-xlink:href="<[$ctrl.getStaticImageUrl($ctrl.pathIconParameters[$index].thumbnailIconUrl)]>"
-                         xlink:href="">
-                  </image>
-                </pattern>
-              </defs>
-              <a xlink:href="<[$ctrl.getExplorationUrl(node)]>"
-                 ng-mouseover="$ctrl.updateExplorationPreview(node);
-                               $ctrl.setIconHighlight(node, $index);"
-                 ng-mouseleave="$ctrl.togglePreviewCard(node);
-                                $ctrl.unsetIconHighlight();"
-                 ng-class="{'oppia-story-node-default-cursor': !$ctrl.isSummaryTileVisible(node)}">
-                <circle ng-show="$ctrl.node.getExplorationId() === $ctrl.nextExplorationId &&
-                  $index !== $ctrl.activeHighlightedIconIndex"
-                        cx="50"
-                        cy="70"
-                        r="50"
-                        id="<['highlight' + $index]>"
-                        fill="#FF8C00"
-                        fill-opacity="0.5"/>
-                <circle ng-show="($index===$ctrl.activeHighlightedIconIndex)?true:false"
-                        cx="50"
-                        cy="70"
-                        r="50"
-                        id="<['highlight' + $index]>"
-                        fill="#A6DACF"
-                        fill-opacity="0.5"/>
-                <circle cx="50"
-                        cy="70"
-                        r="42"
-                        fill="url(#image<[$index]>)"
-                        stroke="#006553"
-                        stroke-width="2"/>
-              </a>
-            </svg>
-          </a>
-        </div>
+                <img ng-if="node.isCompleted()" ng-src="<[$ctrl.getStaticImageUrl('/general/collection_paw.svg')]>" class="lesson-icon-completed oppia-hide-lesson-track">
+                <div ng-if="!node.isCompleted()" class="lesson-icon-uncompleted oppia-hide-lesson-track" />
```

Comment:
I don't think we use self-closing tags in our codebase. Please specify the closing tag here.

Reply:
Done.